compiler/rustc_hir_analysis/src/hir_ty_lowering/mod.rs

```diff
@@ -1988,13 +2091,6 @@ impl<'tcx> dyn HirTyLowerer<'tcx> + '_ {
     }
 
     /// Lower a type from the HIR to our internal notion of a type given some extra data for diagnostics.
```

Comment:
```suggestion
     /// Lower a type from the HIR to our internal notion of a type.
```

(preexisting) this too